resources/profiles/gCreate.ini

```diff
@@ -749,11 +750,16 @@ filament_vendor = ProtoPasta
 temperature = 250
 bed_temperature = 70
 
+[filament:Generic HIPS @GCREATE]
+inherits = *HIPS*

```

Comment:
Invalid inheritance. There is no such section.

Reply:
Oops. I don't have a hips section above. Can we just have it inherit *PET* so it will grab those base settings. Sorry about that. Noted on the version #

Reply:
> Can we just have it inherit PET so it will grab those base settings.

We need some additional changes, like filament_type, density, temperature.

Is it ok like that? Can the printer achieve 100C on the bed? It was set like that in your profile, but I noticed that you are using lower temperature for normal ABS.

```
[filament:Generic HIPS @GCREATE]
inherits = *PET*
filament_vendor = Generic
filament_density = 1.04
temperature = 230
first_layer_temperature = 235
bed_temperature = 100
first_layer_bed_temperature = 100
filament_type = HIPS
```

Reply:
The bed can handle 120C. If it's easier you can just remove hips. I can handle it in a future update. All I really cared about was the jerk speed and start gcode. The jerk speed change dramatically reduce print time.